core-ui/src/components/Predefined/Details/Pod/ContainersLogs.js

```diff
@@ -179,10 +179,10 @@ export const ContainersLogs = ({ params }) => {
         <PageHeader
           title={params.containerName}
           breadcrumbItems={breadcrumbs}
-        ></PageHeader>
+        />
       </div>
       <LayoutPanel className="fd-margin--md logs-panel">
-        <LayoutPanel.Header>
+        <LayoutPanel.Header className="logs-panel-header">
```

Comment:
Wouldn't it be better to have a smaller dropdown (all entries in the dropdown are short) and fir everything into one line?